src/3d/qgs3dmapcanvas.h

```diff
@@ -240,6 +255,9 @@ class _3D_EXPORT Qgs3DMapCanvas : public QWindow
     QString mCaptureFileFormat;
 
     QgsTemporalController *mTemporalController = nullptr;
+
+    //! This holds rubber bands for highlighting identified features
+    QMap<QgsMapLayer *, QgsRubberBand3D *> mHighlights;
```

Comment:
may be good to add info about ownership of pointers.

shall we clear highlights in the destructor?

Reply:
yes, indeed